websites/M/Merriam-Webster/presence.ts

```diff
@@ -0,0 +1,267 @@
+const presence = new Presence({
+		clientId: "1002734097719894067",
+	}),
+	browsingTimestamp = Math.floor(Date.now() / 1000);
+
+presence.on("UpdateData", async () => {
+	const presenceData: PresenceData = {
+			largeImageKey: "https://i.imgur.com/12lTZIs.png",
+			startTimestamp: browsingTimestamp,
+		},
+		{ pathname } = window.location;
+
+	if (document.location.pathname === "/")
+		presenceData.details = "Browsing the dictionary";
+	else if (document.location.pathname.startsWith("/dictionary/")) {
+		presenceData.details = "Viewing definitions";
+		presenceData.state = `${document
+			.querySelector("head > title")
+			.textContent.replace(" Definition & Meaning - Merriam-Webster", "")
+			.replace(
+				" - Definition and More from the Free Merriam-Webster Dictionary",
+				""
+			)}`;
```

Comment:
Look for better selectors that don't require all that replacing.
```suggestion
		presenceData.state = document.querySelector("#left-content > div:nth-child(2) > div:nth-child(1) > h1").textContent
```